core/jazz_services/index.js

```diff
@@ -125,6 +125,7 @@ module.exports.handler = (event, context, cb) => {
                     // fetch services list from dynamodb, filter if required
                     fetchServices: function (onComplete) {
                         var query = event.query;
+                        query['isSearch'] = true;
```

Comment:
@raghits  @dmpl1  Shouldn't this be `true` only when the path is `/services/search`?

Reply:
@devsatishm This is implement in https://github.com/tmobile/jazz/pull/1223

Reply:
@devsatishm @raghits, path check for search is addressed separately in above PR [#1223], Please additional comments related to search  issue is also addressed at #1223.